Makefile.u

```diff
@@ -19,10 +19,9 @@ inc = -I/usr/local/include -Isrc -Isrc/3dgfx -Isrc/rt -Isrc/scr -Isrc/utils \
 	  -Isrc/glut -Ilibs -Ilibs/imago/src -Ilibs/anim/src -Ilibs/mikmod/include \
 	  -Ilibs/goat3d/include
 def = -DMIKMOD_STATIC -DNO_ASM -DMINIGLUT_USE_LIBC -DNDEBUG
-opt = -O3
```

Comment:
The opt and dbg variables here are just an easy way to enable or disable optimizations and debug flags, so let's just set `opt = $(OPT_sys)` here, and keep using `$(opt)` in CFLAGS below